web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -50,12 +52,16 @@ const AutoCalculatedHint = (volume) => {
 
   return (
     <>
-      These limits are affected by:
+      {/* TRANSLATORS: header for a list of items */}
+      {_("These limits are affected by:")}
       <List>
         {volume.snapshotsAffectSizes &&
-          <ListItem>The configuration of snapshots</ListItem>}
+        // TRANSLATORS: list item, this affects the computed partition size limits
+        <ListItem>{_("The configuration of snapshots")}</ListItem>}
         {volume.sizeRelevantVolumes && volume.sizeRelevantVolumes.length > 0 &&
-          <ListItem>Presence of other volumes ({volume.sizeRelevantVolumes.join(", ")})</ListItem>}
+        // TRANSLATORS: list item, this affects the computed partition size limits
```

Comment:
NP: is correctly indented?

Reply:
Yes. Otherwise the `eslint` check would complain and fail. We have quite strict CI for that. :wink: